meta-psa/recipes-security/psa-arch-tests/psa-arch-tests_0.9.bb

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+HOMEPAGE = "https://github.com/ARM-software/psa-arch-tests"
+
+DESCRIPTION = "\
+The Arm Platform Security Architecture Test Suite provides tests \
+for implementations of PSA specifications. \
+"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE.md;md5=2a944942e1496af1886903d274dedb13"
+
+FULL_VERSION = "19.06_API0.9"
+SOURCE_DIR_NAME = "${PN}-${FULL_VERSION}"
+SRC_URI = " \
+    https://github.com/ARM-software/psa-arch-tests/archive/v${FULL_VERSION}.tar.gz \
+    file://0001-Support-absolute-directories-for-build-option.patch \
+    file://0002-Don-t-require-linker-script-when-TEST_COMBINE_ARCHIV.patch \
+    file://0003-Add-support-for-specifying-toolchain-using-environme.patch \
+    file://0004-Add-support-for-targets-that-implement-the-C-standar.patch \
+    file://0005-Don-t-define-32-bit-specific-code-if-we-don-t-need-t.patch \
+    file://main.c;subdir=${SOURCE_DIR_NAME} \
+"
+
+SRC_URI[md5sum] = "4bcb1d70615f433b4a949a58354bb896"
+SRC_URI[sha256sum] = "e69a05e77462ccf78ff91e012df378e4673eb06b94969209adf415a0d2b8238d"
+
+S = "${WORKDIR}/${SOURCE_DIR_NAME}"
+B = "${WORKDIR}/build"
+
+PACKAGECONFIG ??= "mbedcrypto"
+PACKAGECONFIG[mbedcrypto] = ",,mbed-crypto,mbed-crypto"
+
+DEPENDS += "dos2unix-native bash-native"
+
+PARALLEL_MAKE = ""
+TARGET = "tgt_dev_apis_stdc"
+
+# Parse PACKAGECONFIG
+python __anonymous() {
+    configs = d.getVar('PACKAGECONFIG').strip().split()
+
+    for config in configs:
+        if config == "mbedcrypto":
+            d.appendVar("PSA_LIB_OPTIONS", " -lmbedcrypto")
+            d.appendVar("SUITES", " crypto")
+}
+
+do_configure() {
+    # psa-arch-tests usually finds out which PSA APIs are implemented by
+    # reading variables in the makefile for the platform.
+    # The Linux psa-arch-tests platform makefile doesn't know which PSA libs
+    # will be installed on the target, though, so we'll have to set those
+    # variables as required before we build.
+
+    platform_makefile=${S}/api-tests/platform/targets/${TARGET}/Makefile
+
+    for suite in ${SUITES}; do
+        config_var_name=$(echo "PSA_${suite}_IMPLEMENTED" | tr "[a-z]" "[A-Z]")
+        sed -i -e "s/^${config_var_name}:=0/${config_var_name}:=1/" ${platform_makefile}
+    done
+}
+
+do_compile() {
+    # The psa-arch-tests build doesn't provide a main() function so we have to
+    # build one ourselves and link it into each test suite we build
+    compile_main_obj
+    for suite in ${SUITES}; do
+        compile_suite_libs ${suite}
+        link_test_binary ${suite}
+    done
+}
+
+# Compile libraries:
+# 1. test_combine.a (contains code for a suite of tests)
+# 2. pal_nspe.a (contains platform (Linux) specific code)
+# 3. val_nspe.a (contains test runner code and code to glue the tests and the
+#    platform specific code together)
+compile_suite_libs() {
+suite=${1:?}
```

Comment:
> Is there a missing tab here? Comment goes to line 100 of this file too.

I tend not to indent assignment of args to variables in shell scripts to indicate that they're part of the function's interface rather than its implementation (and to document that interface).

That's not consistent with our other code though, so I'll indent.

Reply:
Addressed in https://github.com/ARMmbed/meta-mbl/pull/598/commits/b8fd3c3c9de4f462e37ac77688ce123ae2f060cb